### PR TITLE
Node6hotfixes/azure file copy

### PR DESCRIPTION
**Pull request by agrataru**
**Task name**: AzureFileCopy V2, V3 and V4

**Description**: 
* Updating the node version to node 10 
* Bumping up the minor version

*Testing*

*Windows Agent*

**AzureFileCopyV2**
* Tested the blob upload and VM upload

**AzureFileCopyV3**
* Tested the blob upload and VM upload

**AzureFileCopyV4**
* Tested the blob upload and VM upload

**Documentation changes required:** (Y/N) 

**Added unit tests:** (Y/N)

**Attached related issue:** (Y/N) 

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/AzureFileCopyV3/package.json [hunk 1]
```diff
@@ -1,12 +1,15 @@
 {
   "name": "AzureFileCopyV2",
   "dependencies": {
-    "@types/mocha": "2.2.48",
-    "@types/node": "6.0.68",
+    "@types/mocha": "^5.2.7",
```
Comment (arjgupta) [inline Tasks/AzureFileCopyV3/package.json:4]: You should stick to a specific version. for all the packages.

File: Tasks/AzureFileCopyV3/package.json [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
+    "@types/node": "^10.17.0",
     "@types/q": "1.0.7",
     "azure-pipelines-tasks-azure-arm-rest-v2": "1.0.4",
-    "azure-pipelines-task-lib": "2.8.0",
+    "azure-pipelines-task-lib": "3.0.6-preview.0",
```
Comment (arjgupta) [inline Tasks/AzureFileCopyV3/package.json:8]: is there a stable version of this library that we can use ? Or do we have to use this version ?

**Review by arjgupta — CHANGES_REQUESTED** [review]

**Review by arjgupta — APPROVED** [review]

File: Tasks/AzureFileCopyV2/task.json [hunk 3]
```diff
@@ -13,7 +13,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 178,
+        "Minor": 179,
```
Comment (egor-bryzgalov) [inline Tasks/AzureFileCopyV2/task.json:16]: Could you please bump the version in accordance with the current sprint as per [this guide](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/taskversionbumping.md) for all tasks changed?

File: Tasks/AzureFileCopyV3/package.json (hunk 2 above)
Comment (egor-bryzgalov) [reply Tasks/AzureFileCopyV3/package.json:8]: @arjgupta Hi! This is the version prepared for Node 10 migration.

File: Tasks/AzureFileCopyV2/task.json (hunk 3 above)
Comment (agrataru) [reply Tasks/AzureFileCopyV2/task.json]: done

**Pull request merged**